mathics/builtin/forms.py

```diff
@@ -1,5 +1,11 @@
 """
-Print Forms
+Forms of Input and Output
+
+A <i>Form</i> format specifies the way Mathics Expression input is read or output written.
+
+A list of forms is contained in variable '$OutputForm'.
```

Comment:
Maybe "the variable contains..."?

Reply:
Thanks - all of these should be corrected now, but please double check.